src/Data/Relation/Nullary/Decidable/Ext.agda

```diff
@@ -0,0 +1,15 @@
+{-# OPTIONS --safe #-}
+
+module Data.Relation.Nullary.Decidable.Ext where
+
+open import Level using (Level)
+open import Function.Bundles using (_⇔_; mk⇔; module Equivalence)
+open import Relation.Nullary.Decidable.Core using (Dec; map′)
+private
+  variable
+    a b : Level
+    A B : Set a
+
+map′⇔ : (A ⇔ B) → (Dec A ⇔ Dec B)
+map′⇔ A⇔B = mk⇔ (map′ (to A⇔B) (from A⇔B)) (map′ (from A⇔B) (to A⇔B))
+  where open Equivalence
```

Comment:
An extension of `map'` to make it's application to equivalences simpler (e.g. deriving `∃?-sublist-⇔` from `∃-sublist-⇔` in `Ledger.Gov`, near line 88).